gradle/lib/dependabot/gradle/file_parser.rb

```diff
@@ -298,13 +298,17 @@ def buildfiles
         end
       end
 
+      def self.find_includes(buildfile, dependency_files)
+        buildfile.content.
+          scan(/apply from(\s+=|:)\s+['"]([^'"]+)['"]/).flatten.
```

Comment:
This won't work for the Kotlin DSL as the syntax is different. Also, I don't think this covers the full Groovy syntax that's supported for this declaration.